src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -263,7 +263,7 @@ public BlockGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 	@Override protected void initGUI() {
 		destroyTool.setRenderer(new ItemTexturesComboBoxRenderer());
 
-		blocksToReplace = new MCItemListField(mcreator, ElementUtil::loadBlocks);
+		blocksToReplace = new MCItemListField(mcreator, ElementUtil::loadBlocks, false, true);

```

Comment:
Shouldn't here be loadBlocksAndTags too?

Reply:
@Goldorion this question is still to be resolved

Reply:
Maybe. I didn't check the rest of comments yet